### DISCUSSION TITLE
Right To Left language support

### DISCUSSION THREAD

**Discussion by GodFatherMK** [RafaelGB/obsidian-db-folder#380] [Ideas]
Please make Obsidian-db-folder support Right To Left languages.

**Comment by RafaelGB** [comment]
Hi @GodFatherMK , We need to support any translation, I am agree

**Reply by GodFatherMK** [reply]
Thanks for your support @RafaelGB 

Do you have a specific time range for adding Right To Left Language to Obsidian db-folder Plugin?

**Reply by RafaelGB** [reply]
As a hobby I do not really have deadlines. I am  giving priority to the issues depends of the demand of the users

**Comment by RafaelGB** [comment]
https://github.com/RafaelGB/obsidian-db-folder/issues/386
